packages/clay-modal/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: './src/ClayModal.js',
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /(node_modules|bower_components)/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            compact: false,
+            presets: [
+              [
+                'babel-preset-env',
+                {
+                  targets: {
+                    browsers: ['last 2 versions'],
```

Comment:
It is no longer necessary to use it since we are extending it from `browserslist-config-clay-components`. I forgot to remove 😅

Reply:
Hahaha, I didn't see it 😅